pkg/crd/markers/crd.go

```diff
@@ -268,6 +268,9 @@ func (s Resource) ApplyToCRD(crd *apiext.CustomResourceDefinitionSpec, version s
 	if s.Path != "" {
 		crd.Names.Plural = s.Path
 	}
+	if s.Singular != "" {
+		crd.Names.Singular = s.Singular
+	}
```

Comment:
Can we pl. add a test for this.  pkg/crd has a project in testdata that we are using for tests. 

otherwise, LGTM to me.